app/validators/representation_order_validator.rb

```diff
@@ -40,7 +40,7 @@ def claim
   end
 
   def earliest_permitted
-    if claim.try(:interim?)
+    if claim&.lgfs? && claim&.interim?
```

Comment:
There's quite a lot of changes like this one in this PR, and as stated in the description and the commit message that includes these changes, the main reason behind this is to simply preserve the existent behaviour we had so far (which only applied to **LGFS** interim claims).